stable/signalsciences/Chart.yaml

```diff
@@ -13,7 +13,7 @@ keywords:
 - firewall
 - waf
 sources:
-- https://github.com/marccampbell/docker-sigsci
+- https://github.com/helm/charts/tree/master/stable/signalsciences
```

Comment:
this should points where you can find the code source, not the helm chart

Reply:
i've updated to their github. The dockerfile for this container is not public yet.